tests/test_msstats_in.py

```
@@ -0,0 +1,462 @@
+                intensities_channel_counts[channel] += 1
+
+                # Count file-channel combinations from intensities
+                if ref_file not in file_channel_matrix_from_intensities:
+                    file_channel_matrix_from_intensities[ref_file] = {}
+                if channel not in file_channel_matrix_from_intensities[ref_file]:
+                    file_channel_matrix_from_intensities[ref_file][channel] = 0
+                file_channel_matrix_from_intensities[ref_file][channel] += 1
+
+    # Update the main counting with intensities data
+    if intensities_channel_counts:
+        for channel, count in intensities_channel_counts.items():
```

Ruff (0.12.2)

Loop control variable `count` not used within loop body

(B007)